### DISCUSSION TITLE
Reimplement date time format for CSV Export option in Grafana 7.

### DISCUSSION THREAD

**Discussion by ghost** [grafana/grafana#38389] [Dashboards, Panels & Visualizations]
**What would you like to be added**:
I want to ask about possibility of bringing back date time formatting from old CSV export to new CSV export in Grafana 7.

**Why is this needed**:
Date Time format allowed to change the default timestamp format (adding milliseconds, omitting date part etc.) so the exported data can be used properly. In Grafana 7 I do not see this option in new CSV export. The option was useful because default timestamp is not always the best solution for every use case.

This is how it in in Grafana 6:
![image](https://user-images.githubusercontent.com/31473337/84903854-e326f280-b0ae-11ea-9f5d-74f6c6e52293.png)

This is how it is in Grafana 7, there is no of Date Time format option:
![image](https://user-images.githubusercontent.com/31473337/84903779-cf7b8c00-b0ae-11ea-8a69-cf0c89380a35.png)

**Comment by MarkoBursic** [comment]
I hve the same problem here. The date&time format is wrong for my country, previous was OK. Exported data have large format, while I need only one decimal point up to three. This causes a large CSV files, while the previous version 6 was doing everything OK, now it is gona a step back with new issues.

**Comment by scaarup** [comment]
Same problem here.

**Comment by aleks2a** [comment]
Has anyone found how to make it work?

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82982. Feel free to continue the discussion around this there. Thank you!
